Cargo.toml

```diff
@@ -28,7 +28,7 @@ fnv = "^1.0"
 lazy_static = "^1.4"
 libc = { version = "^0.2", optional = true }
 parking_lot = "^0.12"
-protobuf = { version = "^2.0", optional = true }
+protobuf = { version = "^3.7", optional = true }
```

Comment:
The security fix is actually in `3.7.2`, can we please directly put that as a lower limit in here?

Reply:
good idea, done in aae3a0df414fc6a7fe36a54c773e7d0a0c66ad12